### ISSUE TITLE
Improve search results (list & map)

### ISSUE THREAD

**Issue by muralito** [organicmaps/organicmaps#6442]
For results lists, especially when they are long, it may be useful for the user to be able to filter whether the items are open or closed.

In the current OM version (2023-09) the "Closed" red text appears in the list when the item is closed, but if the user choose to show them in the map all items appears as blue dots, with no indication of which of them are closed.

My proposal is to add the opening hours to the result list (maybe green if there are enough time until closure, and also the opening and closing hours for today), orange if the item is close to the closing hours ("Closes in 20 minutes"), and red if closed, and also add the closed time ("Closed at 18:00" or "Closed Mondays") and/or the next opening hour ("Opens tomorrow at 09:00", or "Opens Tuesday at 12:00")

The same colour schema should be used to display the dots in the map.

**Comment by patepelo** [comment]
It would be nice to refine search by filtering by "Open Now".  That culd be a good first step. Although, at leat in my area, opening times are not very matured data on OSM yet. Maybe in other cities the data is already god enough for incorporating.

**Comment by biodranik** [comment]
Open now is a good start, very useful one. But we should also show "unknown" opening hours.

In the long term it would be easy to add opening hours using OM everywhere. It's just a question of time and a convenient tool to do that (easy OM UX).

**Comment by patepelo** [comment]
> Open now is a good start, very useful one. But we should also show "unknown" opening hours.
> 
> In the long term it would be easy to add opening hours using OM everywhere. It's just a question of time and a convenient tool to do that (easy OM UX).

### Idea: Open questions. 
It would be nice if we could soon experiment with binary questions in OM:

A user is at a restaurant, checks for the details in OM in the place page, a question at the end appears:

"Is this place wheel chair accesible?" or "Are there vegan options in the menu?". 

Similar to StreetComplete but with a visited POI approach.

**Comment by muralito** [comment]
> Open now is a good start, very useful one. But we should also show "unknown" opening hours.

I assume that if the element has not the tag opening_hours, showing nothing like it is now is not a bad choice. But also showing, "unknown opening hours" and "Add them" could help users adding data.

Also an use case like this: the user arrives at a place that info say it should be open, but it's closed. it could be useful to have an option to report or fix the data in an easy way.

> In the long term it would be easy to add opening hours using OM everywhere. It's just a question of time and a convenient tool to do that (easy OM UX).
Please also have in mind the use case of fixing data. It should be the same or almost the same UX to add opening hours, but with the existing data preloaded. And choosing how/when the user is offered to collaborate with opening hours.

**Comment by muralito** [comment]
> It would be nice to refine search by filtering by "Open Now". That culd be a good first step. Although, at leat in my area, opening times are not very matured data on OSM yet. Maybe in other cities the data is already god enough for incorporating.

Yes, data is not equal quality or matured around the globe, and some features have fixed opening_hours forever, and others change more frequently (e.g. restaurants) and it makes it harder to keep their data updated

It is somehow the egg-chicken problem. Seeing that data consumers are using this info, and final users are getting a better quality guidance, undoubtedly will make collaborations more frequent when seeing them useful.

Anyways, for cities that do not have enough data, having this feature does not harm, don't it?
